libc-test/build.rs

```diff
@@ -4056,6 +4056,33 @@ fn test_linux(target: &str) {
                 true
             }
 
+            // FIXME: seems to not be available all the time (from <linux/include/sched.h>:
```

Comment:
```suggestion
            // FIXME: seems to not be available all the time (from <include/linux/sched.h>:
```

Seems swapped, but yeah, it's weird that 23.10 couldn't find them which has the v6.5 kernel header 🤔